src/background.ts

```diff
@@ -50,7 +62,7 @@ const gardenInspect = (tabId?: number, toggle?: boolean): void => {
   }
```

Comment:
I think the function signature for [`chrome.storage.local.get` requires passing in `null` as the first parameter.](https://developer.chrome.com/docs/extensions/reference/storage/#method-StorageArea-get)

Reply:
The `get` function provides [multiple signatures](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/86fbf93e9dc4282914d59abdde8a0503543be1df/types/chrome/index.d.ts#L7420-L7441). The current use is valid (and the extension works as expected with `yarn start`).

Reply:
Ahhh good find.